tutorials/shading/migrating_to_godot_shader_language.rst

```diff
@@ -152,7 +152,7 @@ of helpful information from the main program into the fragment shader for you. A
 have no equivalents in Godot because Godot has chosen not to make them available by default.
 This is okay because Godot gives you the ability to make your own uniforms. For variables whose
 equivalents are listed as "Provide with Uniform", the user is responsible for creating that
-uniform themself. The description gives the reader a hint about what they can pass in as a substitute.
+uniform himself. The description gives the reader a hint about what they can pass in as a substitute.
```

Comment:
"themselves" is the form we want here, as we want a gender-neutral theme :slightly_smiling_face:

"themself" isn't a word though, so we should replace it with "themselves".